src/App.js

```diff
@@ -1,16 +1,34 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
 const App = () => {
+  const [messages, setMessages] = useState(chatMessages);
+
+  const totalLikes = messages.reduce((total, entry) => {
+    total += entry.liked;
+    return total;
+  }, 0);
+
+  const setLike = (id) => {
```

Comment:
A mior nitpick from me here is that the name `setLike` sounds like it will always be setting the liked status to true. (This would feel different if it also accepted a value to use for the setting, not just the id of the message to "set", but we don't want to do that. We want the logic about whether to set or not to be contained here). As a result, I'd rather rather see this named something like `updateLike` or `toggleLike`.